pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[project]
+name = "newsscraper"
```

Comment:
ok I added `uv` which is an NPM like package manager for python. so I used this instead of pip. This is now the industry standard. but it doesn't change anything on your side. you can use pip like you used to before and ignore this file.

added the instructions in readme.